src/compo/tropomi_no2_co_nc2ioda.py

```diff
@@ -77,6 +77,8 @@ def _read(self):
             times = np.empty_like(qa_value, dtype=object)
             qa_value = qa_value.ravel()
             nlevs = ncd.groups['PRODUCT'].dimensions['layer'].size
+            cf_value = ncd.groups['PRODUCT'].groups['SUPPORT_DATA'].groups['DETAILED_RESULTS'].\
+                variables['cloud_fraction_crb_nitrogendioxide_window'][:].ravel()
```

Comment:
@jeromebarre Do you know if this is the correct variable to use? Also, I cannot find a similar variable in the co file.